internal/servers/controller/internal/metric/api.go

```diff
@@ -227,7 +227,7 @@ func InstrumentApiHandler(wrapped http.Handler) http.Handler {
 		promhttp.InstrumentHandlerDuration(
 			httpRequestLatency.MustCurryWith(l),
 			promhttp.InstrumentHandlerRequestSize(
-				httpResponseSize.MustCurryWith(l),
```

Comment:
Looks like you need to update the PR since this is already reflected in the main branch.